persian/otagheh-basteh.md

```diff
@@ -0,0 +1 @@
+چشم‌هایم را در یک اتاق کوچک و بسته باز کردم. نگاهی به اطراف انداختم. نمی‌دانستم کجا هستم و چگونه وارد این اتاق شده‌ام. هیچ در و پنجره‌ای در این اتاق کوچک ندیدم و هیچ صدایی نمی‌شنیدم.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.